pkg/windows/windows.go

```diff
@@ -591,9 +597,32 @@ func (vm *windows) createDirectories() error {
 // removeDirectories removes all directories created as part of the configuration process
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
+	existingFiles, err := getFilesToTransfer()
+	if err != nil {
+		return errors.Wrapf(err, "error getting list of files transferred to the instance")
+	}
+	for src, dest := range existingFiles {
+		filePath := dest + "\\" + filepath.Base(src.Path)
```

Comment:
Shouldn't you be using `path.Join` or `filepath.Join` or does that not work given these are Windows filepaths?

Reply:
This is the pattern used in this file. I looked into it, "path/filepath" can be used as it is OS aware.

Reply:
>This is the pattern used in this file.

That does not make it correct though and we shouldn't be adding to it.

>I looked into it, "path/filepath" can be used as it is OS aware.

Aren't you constructing a Windows path on a Linux OS? So maybe that is why filepath.Join() was not used.

Reply:
I agree, we should not follow the pattern. I was stating the fact, I am testing an alternative, will update.

Reply:
I tested it out, current way is the only way to proceed. As you mentioned we are creating a path for Windows on linux using an inbuilt function is not feasible here.